proto/cosmos/msg/v1/msg.proto

```diff
@@ -27,4 +27,61 @@ extend google.protobuf.MessageOptions {
   // kind in case the signer information is contained within
   // a message inside the cosmos message.
   repeated string signer = 11110000;
+
+  // legacy_amino_name is the string used when registering a concrete
+  // type into the Amino type registry, via the Amino codec's
+  // `RegisterConcrete()` method. This string MUST be at most 39
+  // characters long, or else the message will be rejected by the
+  // Ledger hardware device.
+  string legacy_amino_name = 11110001;
+
+  // legacy_amino_encoding describes the encoding format used by Amino
+  // for the message. The field type is chosen to be a string so that
+  // the value can either be:
+  // - a machine-readable string, such as "base64", "bech32" or "utf8",
+  // - or a human-readable string, for instance a short specification of how
+  // a big integer would be encoded using Amino.
```

Comment:
Does it make sense to have two separate annotations for human and machine readable?

We could also have all options under a struct, i.e. `LegacyAminoMessageOptions legacy_amino = 110001`

Reply:
Dan was proposing to use proto comments for the human-readable description, which I think is a good idea

Reply:
I like the struct idea, the resulting annotations length is anyways the same

Reply:
Does this actually work? Doing this, I get `unknown extension cosmos.msg.v1.legacy_amino.name`, which makes sense: how do you distinguish between the `legacy_amino` message option in the v1 option, and potentially another `name` message option in another `legacy_amino` subpackage.